src/components/AvatarGroup/AvatarGroup.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import Avatar from "../Avatar/Avatar";
+import Counter from "../Counter/Counter";
+import Tooltip from "../Tooltip/Tooltip";
+import AvatarGroupCounter from "./AvatarGroupCounter";
+import styles from "./AvatarGroup.module.scss";
+
+const AvatarGroup = ({
+  className,
+  avatarClassName,
+  id,
+  children,
+  size,
+  type,
+  max,
+  counterProps,
+  counterTooltipCustomProps
+}) => {
+  if (!children) {
+    return null;
+  }
+
+  children = React.Children.toArray(children);
```

Comment:
it's weird that esline don't give u error for this. it's not standard to override prop value under the same name.
```suggestion
  const overrideChildren = React.Children.toArray(children);
```